posts/openscanhub-prototype/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "SAST using OpenScanHub is here!"
+date: 2024-07-30T08:46:18+02:00
+authors: lbarczio
+tags:
+  - copr
+  - srpm
+  - sast
+  - openscanhub
+---
+
+We are excited to announce a new experimental feature in our service: the integration of Static
+Application Security Testing (SAST) using [OpenScanHub](https://openscanhub.fedoraproject.org/).
+
+OpenScanHub is a service that runs various static analyzers on RPM packages - by default `Cppcheck`,
+`ShellCheck`, the static analyzers embedded in `GCC` and `Clang`, and the `find-unicode-control` tool.
+
+Let's have a look at the functionality we are bringing with this!
+
+<!--truncate-->
+
```

Comment:
Can we have a "why" section here? We need to tell users why this feature is enabled for `rawhide`. May be my talk at [DevConf](https://www.youtube.com/watch?v=rcuIvAttWgY) can give some context here.

Reply:
added, PTAL